### ISSUE TITLE
Sync sublist of observable list after adding element in base list

### ISSUE THREAD

**Issue by manish5196** [canoo/dolphin-platform#726]
Add element in the sublist of the observable list when adding an element in the base list. #721

**Comment by hendrikebbers** [comment]
@kunsingh @manish5196 is this still open?

**Comment by kunsingh** [comment]
Yes it is open....But why do we want to sync? Java does not sync and its a challenge also specially in case of addAll and retainAll.

**Comment by hendrikebbers** [comment]
So this is currently working as Java is working in default?

**Comment by kunsingh** [comment]
Let me look deeply...will get back on this.

**Comment by hendrikebbers** [comment]
Ach the moment the list has the same behaviour as the ArrayList in Java. Based on the observable functionality we can add a real sync later. Added Future tag.
